src/main/webapp/app/entities/programming-exercise/programming-exercise.component.html

```diff
@@ -142,6 +142,15 @@ <h4>
                                     <fa-icon [icon]="'pencil-alt'"></fa-icon>
                                     <span class="d-none d-md-inline" jhiTranslate="entity.action.editInEditor">Edit in Editor</span>
                                 </button>
+                                <button
+                                    type="submit"
```

Comment:
We come to the point where we get too many buttons in the list.
Can you move some of the more specific buttons into the detail component?

Reply:
I think this is not a good idea, because then we need the buttons both in the programming-exercise-link and programming-exercise-update forms.
This would then also necessitate a "go back" functionality which is not implemented atm.

I think the buttons need to be in the programming-exercise.component but we need a better concept on how to display them in the overview, which is not in scope of this PR.

Reply:
I don't think that this would "blow up the changed files", but it's fine for me doing this in another PR. Let's discuss this in our next meeting